<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KOCWX21981" failures="1" tests="2" name="ErrorValidationTest" time="13.759" errors="0" timestamp="2022-11-11T11:00:44 IST">
  <testcase classname="AutomationTraining.Tests.ErrorValidationTest" name="LoginErrorValidation" time="0.577"/>
  <testcase classname="AutomationTraining.Tests.ErrorValidationTest" name="submitOrder" time="2.256">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at AutomationTraining.Tests.ErrorValidationTest.submitOrder(ErrorValidationTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
</testsuite> <!-- ErrorValidationTest -->
